src/highdicom/map/sop.py

```diff
@@ -0,0 +1,789 @@
+        content_description: Optional[str] = None,
+        content_creator_name: Optional[str] = None,
+        pixel_measures: Optional[PixelMeasuresSequence] = None,
+        plane_orientation: Optional[PlaneOrientationSequence] = None,
+        plane_positions: Optional[Sequence[PlanePositionSequence]] = None,
+        **kwargs,
+    ):
+        """
+        Parameters
+        ----------
+        source_images: Sequence[pydicom.Dataset]
+            Sequence of one or more `pydicom.Dataset`s that this parametric map
+            is derived from
+        pixel_array: numpy.ndarray
+            Array of parametric map pixel data of unsigned integer or
+            floating-point data type representing one or more frames of the
+            parametric map pixel data. The values are supposed to represent a
+            single "feature", i.e., be the result of one set of image
+            transformations such that the same `real_world_value_mappings`
+            apply.
+        series_instance_uid: str
+            UID of the series
+        series_number: Union[int, None]
+            Number of the series within the study
+        sop_instance_uid: str
+            UID that should be assigned to the instance
+        instance_number: int
+            Number that should be assigned to the instance
+        manufacturer: str
+            Name of the manufacturer (developer) of the device (software)
+            that creates the instance
+        manufacturer_model_name: str,
+            Name of the model of the device (software)
+            that creates the instance
+        software_versions: Union[str, Tuple[str]]
+            Versions of relevant software used to create the data
+        device_serial_number: str
+            Serial number (or other identifier) of the device (software)
+            that creates the instance
+        contains_recognizable_visual_features: bool
+            Whether the image contains recognizable visible features of the
+            patient
+        real_world_value_mappings: Sequence[highdicom.map.RealWorldValueMapping]
+            Descriptions of how stored values map to real-world values. The
+            concept of real-world values is a bit fuzzy and the mapping may be
+            difficult to describe (e.g., in case of the feature maps of a deep
+            convolutional neural network model).
+        window_center: Union[int, float, None], optional
+            Window center for rescaling stored values for display purposes by
+            applying a linear transformation function. For example, in case of
+            floating-point values in the range ``[0.0, 1.0]``, the window
+            center would be ``0.5``, in case of floating-point values in the
+            range ``[-1.0, 1.0]`` the window center would be ``0.0``, in case
+            of unsigned integer values in the range ``[0, 255]`` the window
+            center would be ``128``.
```

Comment:
My understanding is that this is just a suggestion that viewers can ignore and users of viewers can alter anyway. I would change the language from "would be" to "could be" or "may be". There may be good reasons to choose a window center somewhere other than the center of the full range, if for example most of interesting detail exists within a certain range of values. Similarly for window width below

Reply:
Addressed via 351475c61e7f58446485b552347ed8d54c6f2479